vaadin-flow-components-shared-parent/vaadin-flow-components-test-util/src/main/java/com/vaadin/tests/ThemeVariantTestHelper.java

```diff
@@ -0,0 +1,28 @@
+package com.vaadin.tests;
+
+import com.vaadin.flow.component.HasThemeVariant;
+import com.vaadin.flow.component.ThemeVariant;
+import org.junit.Assert;
+
+import java.util.Set;
+
+public class ThemeVariantTestHelper {
```

Comment:
Extracted a helper for testing adding / removing theme variants for component implementing the new interface. There might be a better pattern for extracting test logic. What I wanted to avoid is sub-classing, that feels too opaque and some tests might already extend a different class.